modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/serieController.js

```diff
@@ -22,11 +22,20 @@
 
 // Controller for all single series screens.
 angular.module('adminNg.controllers')
-.controller('SerieCtrl', ['$scope', 'SeriesMetadataResource', 'SeriesEventsResource', 'SeriesAccessResource',
-  'SeriesThemeResource', 'ResourcesListResource', 'UserRolesResource', 'Notifications', 'AuthService',
-  'StatisticsReusable',
-  function ($scope, SeriesMetadataResource, SeriesEventsResource, SeriesAccessResource, SeriesThemeResource,
-    ResourcesListResource, UserRolesResource, Notifications, AuthService, StatisticsReusable) {
+.controller('SerieCtrl', ['$scope', 'SeriesMetadataResource',
+  'SeriesEventsResource', 'SeriesAccessResource', 'SeriesThemeResource',
```

Comment:
As much as the previous formatting is horrible, I would avoid reformatting it...

Reply:
I had to. The maxiumum length of 120 characters was reached in Eslint.